africanus/model/spi/tests/test_component_spi.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""Tests for `codex-africanus` package."""
+
+import numpy as np
+import pytest
+
+
+def test_vs_scipy():
+    """
+    Here we just test the per component spi fitter against
+    a looped version of scipy's curve_fit
+    :return:
+    """
+    from africanus.model.spi import fit_spi_components
+    from scipy.optimize import curve_fit
```

Comment:
scipy isn't a base dependency, so we should do the following.

```suggestion
curve_fit = pytest.importorskip('scipy.optimize').curve_fit
```